com.bitwig.BitwigStudio.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.bitwig.BitwigStudio</id>
+  <metadata_license>CC0</metadata_license>
```

Comment:
I'm sorry I'm not familiar with this file, what does mean license CC0?
Is there a spec somewhere?

Reply:
It is creative commons CC0 https://creativecommons.org/share-your-work/public-domain/cc0/. This is the license of _this file_ and not the software.

CC0 is one of the few acceptable to submit to flathub.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license

Reply:
If we can make it MIT, it would be better for us; because we are not lawyer and MIT is so simple, it is one that we like and understand without effort at Bitwig ;-)

Reply:
MIT is for code. This is data.

Also https://github.com/flathub/flathub/wiki/App-Requirements#appstream

>  If application metadata has not been provided by the upstream, it should be licensed with Creative Commons Zero, version 1, by stating CC0-1.0 in metadata_license.

There is no effort. CC0 is public domain, be specified as a lot of jurisdiction don't recognize Public Domain except when copryight expired (most if not all Europe for example)

Reply:
But I realize I have to write "CC0-1.0"